.github/actions/run_tests/action.yml

```diff
@@ -46,8 +46,5 @@ runs:
         pip list
         cd tests
         coverage run --rcfile=${{ inputs.coverage_config }} testing_main.py
-        coverage html
-        coverage report
-        coverage-badge -o htmlcov/coverage.svg
-        coverage run --rcfile=${{ inputs.coverage_config }} -m pytest pytest_testing_cosserat_curve.py
-        coverage report
+        cd ..
+        pytest
```

Comment:
Should we then set the testing flags via an `inputs:` variable? Something like `pytest_flags`. That could then simply be set in the workflows. This would make the `require_*` and the `coverage_config` flags obsolete.

Does it make sense to do this setup already in this PR? I think yes, but I will leave that up to you.

Reply:
One thing I have not yet considered is the following:

We have to adjust for the coverage in different pipeline runs. If the performance tests should not include the standard tests the coverage will be lower. Therefore, we somehow have to adjust the necessary coverage for differing runs.

In my opinion we could also just execute the standard tests in every config so the minimum coverage is reached.